app/controllers/offer_controller.rb

```diff
@@ -0,0 +1,29 @@
+class OfferController < ApplicationController
+	def index
+		@offer = Offer.all
+	end
+
+	def show
+		@offer = Offer.find(params[:id])
+	end	
+
+	# def new
+	# 	@offer = Offer.new
+	# 	@product = Product.find(params[:id])
+	# end
+
+	def create
+		@offer = Offer.new(offer_params)
```

Comment:
Czy widok `new` wyświetli błędy walidacji? To opcjonalne, ale całkiem potrzebne

Zamieniłbym też taby na dwie spacje wcięcia (spójność)